docs/section5/Rule5-1.md

```diff
@@ -20,44 +20,42 @@
 
 - **Applicability Check 1:** ```if (rule-5-7  == PASS):```  
 
-- Get building rotation status in the Baseline model: ```is_rotated_baseline = B_RMR.building.is_rotated```  
+- Get building rotation status in the Baseline model: ```is_rotated_b = B_RMR.building.is_rotated```  
 
-- For each building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
+- For each building segment in the Baseline model: ```for building_segment_b in B_RMR.building.building_segments:```  
 
-  - For each thermal_block in building segment: ```thermal_block_baseline in building_segment_baseline.thermal_blocks:```  
+  - For each thermal_block in building segment: ```thermal_block_b in building_segment_b.thermal_blocks:```  
   
-  - For each zone in thermal block: ```zone_baseline in thermal_block_baseline.zones:```  
+  - For each zone in thermal block: ```zone_b in thermal_block_b.zones:```  
 
-  - For each space in zone: ```space_baseline in zone_baseline.spaces:```  
+  - For each space in zone: ```space_b in zone_b.spaces:```  
 
-    - For each surface in space: ```for surface_baseline in space_baseline.surfaces:```  
+    - For each surface in space: ```for surface_b in space_b.surfaces:```  
 
-      - Check that surface is exterior and vertical: ```if ( surface_baseline.adjacent_to == "AMBIENT" ) AND ( 60 <= _surface.tilt <= 90 ): exterior_vertical_surface = surface_baseline```  
+      - Check that surface is exterior and vertical: ```if ( surface_b.adjacent_to == "EXTERIOR" ) AND ( 60 <= _surface.tilt <= 90 ): exterior_vertical_surface_b = surface_b```  
 
```

Comment:
instead of checking the surface type, reference the function for defining surface type.